ADOPTERS.md

```diff
@@ -46,6 +46,7 @@ SPIFFE and SPIRE have integrations available with a number of open-source projec
 * [Emissary](https://github.com/github/emissary)		
 * [Envoy](https://github.com/envoyproxy/envoy)		
 * [Ghostunnel](https://github.com/square/ghostunnel)		
+* [gRPC](https://pkg.go.dev/github.com/spiffe/go-spiffe/v2/examples/spiffe-grpc) 
```

Comment:
I would probably link to https://github.com/grpc/grpc instead of the go-spiffe example. That would also keep consistency with the way that other projects are linked.

Reply:
I was thinking it's more relevant for those exploring existing integrations to be provided with direct links to examples or docs of the integration. Like "Cool, you're letting me know gRPC adopts spiffe, but show me where". Doing so does require updating the other links over time for consistency. What do you think?

Reply:
For clarification, gRPC itself has _very minimal_ support for SPIFFE. The Go implementation only goes so far as to lift the SPIFFEID from the peer certificate and make it available in the [TLSInfo](https://github.com/grpc/grpc-go/blob/712e8d4f57fd4a4fbb83406148f9c71eb3e7714e/credentials/tls.go#L39) which is obtainable by users. It does not support SPIFFE authentication first hand.

go-spiffe leverages existing hooks (i.e. dialing and server options) to provide a TLS configuration to grpc-go that implement SPIFFE authentication.